src/common/flux/action.ts

```diff
@@ -12,21 +12,22 @@ export class Action<TPayload> {
     private listeners: ((payload: TPayload) => void)[] = [];
     private scope: string = 'DEFAULT_SCOPE';
 
-    public invoke(payload: TPayload): void {
-        if (Action.executingScopes[this.scope]) {
+    public invoke(payload: TPayload, scope: string = null): void {
+        const activeScope = scope ? scope : this.scope;
```

Comment:
This can use https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
Done